tests/ttnn/unit_tests/test_multi_device.py

```diff
@@ -65,6 +65,32 @@ def test_multi_device_open_close_using_context_manager(silicon_arch_name, silico
         pass
 
 
+def test_multi_device_open_close_galaxy_mesh(silicon_arch_name, silicon_arch_wormhole_b0):
+    if ttnn.get_num_devices() < 32:
```

Comment:
could we add a pytest galaxy fixture as well so it'll be more accessible for model folks?

Reply:
Maybe we can upgrade the `device_mesh` fixture to accept a DeviceGrid, instead of just num_devices_requested

Reply:
`pytest` doesn't really have a good way to define parametrization on fixtures. It goes against what a fixture is supposed to be. You can see instances of how it's used today if you grep `indirect=True` in the codebase.